test/utils/deepFreeze.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
 export const deepFreeze = (object: any) => {
```

Comment:
Instead of disabling eslint, would sticking an `any` return type on this function achieve the same thing? 

```suggestion
export const deepFreeze = (object: any): any => {
```

Reply:
`object: any ` is still a lint trigger, don't think we have a way to resolve that